Source/Core/Core/PowerPC/Interpreter/Interpreter_FloatingPoint.cpp

```diff
@@ -11,12 +11,110 @@
 #include "Core/PowerPC/Interpreter/Interpreter_FPUtils.h"
 #include "Core/PowerPC/PowerPC.h"
 
-// Extremely rare - actually, never seen.
-// Star Wars : Rogue Leader spams that at some point :|
-void Interpreter::Helper_UpdateCR1()
+namespace
 {
-  PowerPC::SetCRField(1, (FPSCR.FX << 3) | (FPSCR.FEX << 2) | (FPSCR.VX << 1) | FPSCR.OX);
+// Apply current rounding mode
+enum class RoundingMode
```

Comment:
Since you cast from `FPSCR.RN` to this enum type, should this have explicit values (just to make sure we're getting it right; even though `Nearest` is indeed zero in the old code without enum)?